README.md

```diff
@@ -49,7 +49,9 @@ Alternatively, there are instructions for [manual installation].
 
 ## First run and use
 
-To test Arduino-Lufa, open the LUFA_DualVirtualSerial example and click __Verify__. (See Note on __Upload__ below)
+To test Arduino-Lufa, open the LUFA_DualVirtualSerial example, select your board type from
+the `Tools > Board > Arduino LUFA AVR Boards` submenu and click __Verify__. 
+(See Note on __Upload__ below)
```

Comment:
I put a placeholder comment for now, but if we know which are the incompatible boards it wouldn't be too hard to just strip them off the "boards.txt" file so that they don't appear in the LUFA submenu